libnopegl/src/distmap.c

```diff
@@ -474,7 +474,7 @@ int ngli_distmap_finalize(struct distmap *s)
         return NGL_ERROR_INVALID_USAGE;
     }
 
-    const int32_t nb_shapes = (int32_t)ngli_darray_count(&s->shapes);
+    const size_t nb_shapes = ngli_darray_count(&s->shapes);
```

Comment:
I feel like we should probably add a guard in `ngli_distmap_add_shape()` to refuse adding more than `INT32_MAX` shapes